src/index.js

```diff
@@ -20,6 +20,10 @@ if (typeof process === 'undefined' || !process.env?.RTL_SKIP_AUTO_CLEANUP) {
     teardown(() => {
       cleanup()
     })
+  } else {
+    console.warn(
+      `The current test runner does not support afterEach/teardown hooks. This means we won't be able to run automatic cleanup and you should be calling cleanup() manually.`,
```

Comment:
We should add instructions on how to fix this warning i.e. import from `/pure` instead or better, ensure that you always resolve to `/pure` in your test environment.